packaging/windows/generatemsi.ps1

```diff
@@ -144,18 +213,35 @@ if(-Not (RunCandle))
     Exit -1
 }
 
+if(-Not (RunCandleForBundle))
+{
+    Exit -1
+}
+
 if(-Not (RunLight))
 {
     Exit -1
 }
 
+if(-Not (RunLightForBundle))
+{
+    Exit -1
+}
+
 if(!(Test-Path $DotnetMSIOutput))
 {
     throw "Unable to create the dotnet msi."
     Exit -1
 }
 
+if(!(Test-Path $DotnetBundleOutput))
+{
+    throw "Unable to create the dotnet bundle."
+    Exit -1
+}
+
 Write-Host -ForegroundColor Green "Successfully created dotnet MSI - $DotnetMSIOutput"
```

Comment:
Bundles wrap one or more MSIs (one in this case) and provide functionality to install them all as a single unit. On top of the ability to install multiple packages, the bundle engine (Burn) does a lot of small things that can greatly improve the success rate of installations (that MSI doesn't do for itself).

So, all in all, Bundles are awesome but they still need packages to install anything.